middle_end/inline_and_simplify_aux.mli

```diff
@@ -253,6 +253,12 @@ module Env : sig
 
   (** Print a human-readable version of the given environment. *)
   val print : Format.formatter -> t -> unit
+
+  (** Debuginfo to add to call-sites when inlining a function body. *)
```

Comment:
I think these two comments could do with some improvement.  The first one should note the fact that it concatenates the supplied debuginfo with the debuginfo already in the environment.  The second should be more explicit that it's adding information about the current inlining stack to the given debuginfo value.
Also, they're not really specific to call sites, so maybe drop that.